### PR TITLE
Adding work and aboutme section mobile version

### PR DESCRIPTION
**Pull request by ichsansandy**
## Work Section
- Using Grid to adjust the cards
- Using Flexbox for layouting inside cards 

## Aboutme Section
- Using Flexbox both in container also inside cards
- Have issue with the background can't copy from Figma (Only bottom logo included, the top image missing)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -71,12 +71,283 @@ <h2 class="primary-text dark center-text">
...
-                d="M0.292893 0.292893C0.683417 -0.0976311 1.31658 -0.0976311 1.70711 0.292893L7 5.58579L12.2929 0.292893C12.6834 -0.0976311 13.3166 -0.0976311 13.7071 0.292893C14.0976 0.683417 14.0976 1.31658 13.7071 1.70711L7.70711 7.70711C7.31658 8.09763 6.68342 8.09763 6.29289 7.70711L0.292893 1.70711C-0.0976311 1.31658 -0.0976311 0.683417 0.292893 0.292893Z"
-                 />
+                d="M0.292893 0.292893C0.683417 -0.0976311 1.31658 -0.0976311 1.70711 0.292893L7 5.58579L12.2929 0.292893C12.6834 -0.0976311 13.3166 -0.0976311 13.7071 0.292893C14.0976 0.683417 14.0976 1.31658 13.7071 1.70711L7.70711 7.70711C7.31658 8.09763 6.68342 8.09763 6.29289 7.70711L0.292893 1.70711C-0.0976311 1.31658 -0.0976311 0.683417 0.292893 0.292893Z" />
             </svg>
           </button>
         </div>
       </div>
     </section>
+    <section class="works flex-center">
+      <div class="section-container">
```
Comment (Zuhaib042) [inline index.html]: - [ ] Kindly consider using `display: grid` in this main container as suggested in the requirements. It is necessary to use both grid and flex in this milestone with the sequence suggested in the requirements below
  -  In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

File: index.html [hunk 2]
```diff
@@ -71,12 +71,283 @@ <h2 class="primary-text dark center-text">
...
+            <div class="card-image"></div>
+            <div class="card-info">
+              <h4 class="card-title title-text center-text dark">Multi-Post Stories Gain+Glory</h4>
+              <ul class="tag-container">
+                <li class="tag flex-center">HTML</li>
+                <li class="tag flex-center">CSS</li>
+                <li class="tag flex-center">Javascript</li>
+                <li class="tag flex-center">Ruby on Rails</li>
+              </ul>
+              <button class="button card-button white">See project</button>
```
Comment (Zuhaib042) [inline index.html:95]: - [ ] This button is following the styling guidelines when hovering, but it is not following the styling guidelines in the pressed state. you can achieve that using `focus` pseudoclass. kindly have a look at the pressed state guidelines. kindly consider fixing this in all the buttons.
- [ ] Also kindly consider increasing the top and bottom padding in your buttons to make it identical to the figma design which is a requirement.
qw

File: index.html [hunk 3]
```diff
@@ -71,12 +71,283 @@ <h2 class="primary-text dark center-text">
...
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="flex-center flex-column align-center">
+      <div class="aboutme section-container flex-center flex-column align-center">
```
Comment (Zuhaib042) [inline index.html:180]: - [ ] Kindly consider adding the decoration background image seen in the figma design to make it look similar to the figma design.
your design
fd

figma design
as

File: index.html [hunk 4]
```diff
@@ -71,12 +71,283 @@ <h2 class="primary-text dark center-text">
...
+        <h2 class="primary-text dark center-text margin-zero">About Me</h2>
+        <p class="support-text dark center-text">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <button class="button get-resume-button white">Get my resume</button>
+      </div>
+      <div class="aboutme-box-container section-container">
```
Comment (Zuhaib042) [inline index.html]: - [ ] Kindly consider using `ul` for these categories in `About myself` section, then languages, frameworks and skills would be list items `li`. you can take the whole card as a `li`. 
  -  Please use ul and li for categories in "About myself" section. Languages, Frameworks, and Skills should be list items.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @ichsansandy 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Correct Github flow
✅ Descriptive PR


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] Kindly consider writing commit messages in Imperative tense so instead of writing `Removing unused assets`, you can write `Remove unused assets` following best practices.
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ichsansandy) [reply index.html:81]: Hi @Zuhaib042 
Kindly could you check line 85 
I already use grid in the main container that contains all of the cards line 85
Thanks for considering

File: index.html (hunk 3 above)
Comment (ichsansandy) [reply index.html:180]: It has bug that we can't take the background from figma and everyone have issue with it
could you give me advise with this thanks

**Review by jmonto55 — APPROVED** [review]
# STATUS: APPROVED 🟢

Hi @ichsansandy , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

AgreeSealGIF

## Highlights

- Excellent implementation of utility classes like .grid, and .flex ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**